cloud/pkg/controller/manager/location.go

```diff
@@ -113,18 +115,18 @@ func (lc *LocationCache) SecretNodes(namespace, name string) (nodes []string) {
 
 //IsEdgeNode checks weather node is edge node or not
 func (lc *LocationCache) IsEdgeNode(nodeName string) bool {
-	for _, node := range lc.edgeNodes {
+	for _, node := range lc.EdgeNodes {
```

Comment:
code as follow if we use sync.Map:
```
_, ok := lc.EdgeNodes.Load(nodeName)
return ok
```

Reply:
Used map instead of sync.Map because of range issue mentioned above